projects/core/src/anonymous-consents/connectors/anonymous-consent-templates.connector.ts

```diff
@@ -14,9 +14,6 @@ export class AnonymousConsentTemplatesConnector {
   }
 
   loadAnonymousConsents(): Observable<AnonymousConsent[]> | null {
```

Comment:
this one should remove `null` from return type

Reply:
fixed nice catch